package-lock.json

```diff
@@ -1,8 +1,5821 @@
 {
   "name": "@shift72/core-template",
-  "version": "0.1.0-alpha.1",
-  "lockfileVersion": 1,
+  "version": "0.1.0-alpha.2",
```

Comment:
What's all this about? lol

Reply:
Probably gets autoupdate when pulling changes or running npm i. It's the npm core package version

Reply:
There's almost 6000 lines changed in this file... me and Curtis' PRs didn't have any changes to this file (or main.js.map).  What are you doing differently?

Reply:
Main.js.map would get rendered when there' any change. Can be just ignored or I can put it in gitignore. I think the last release is the cause of updates in the package-lock.json. It would work if you test locally.